src/main/java/it/gov/pagopa/payhub/activities/dto/treasury/TreasuryDTO.java

```diff
@@ -0,0 +1,59 @@
+package it.gov.pagopa.payhub.activities.dto.treasury;
+
+import lombok.*;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+/**
+ * DTO for the TreasuryDto
+ */
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class TreasuryDTO {
```

Comment:
@marcomatteuccieng are you renaming fields of Treasury and/or PaymentsReporting (IUV, IUR, IUF)?
If yes, could we align them here?